dotrix_core/src/renderer.rs

```diff
@@ -110,6 +110,13 @@ impl Renderer {
         self.loaded = false;
     }
 
+    /// Drop the backend pipeline for a shader
+    ///
+    /// This should be called when a shader is changed.
+    pub fn drop_shader_pipeline(&mut self, shader: Id<Shader>) {
```

Comment:
to match Backend API I suggest to call it `drop_pipeline`

Reply:
lol that was my original name, but I couldn't decide anyways I'll change it back